pkg/bff/server.go

```diff
@@ -384,3 +376,23 @@ func (s *Server) GetURL() string {
 	defer s.RUnlock()
 	return s.url
 }
+
+// Set the directory service client for testing purposes.
+func (c *GDSClient) SetGDS(client gds.TRISADirectoryClient) {
+	c.gds = client
+}
+
+// Set the members service client for testing purposes.
+func (c *GDSClient) SetMembers(client members.TRISAMembersClient) {
+	c.members = client
+}
+
+// Get the directory service client for testing purposes.
+func (c *GDSClient) GetGDS() gds.TRISADirectoryClient {
+	return c.gds
+}
+
+// Get the members service client for testing purposes.
+func (c *GDSClient) GetMembers() members.TRISAMembersClient {
+	return c.members
+}
```

Comment:
For tests, I think it would be my preference to either have a test `GDSClient` defined in the Tests package that I could set with the interface (e.g. it would be a wrapper for the bufconn connections to the universal mocks). Or I'd prefer to have a `ConnectDirectory` and `ConnectMembers` method on the `GDSClient` that took DialOptions that could have the bufconn dialer passed to. That would eliminate the need for these accessors, which I think make the tests a bit more verbose than necessary.